python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -26,6 +25,7 @@
 from Ganga.GPIDev.Schema import ComponentItem, FileItem, GangaFileItem, Schema, SimpleItem, Version
 from Ganga.Runtime.spyware import ganga_job_submitted
 from Ganga.Utility.Config import ConfigError, getConfig
+from Ganga.Utility.external.OrderedDict import OrderedDict as oDict
```

Comment:
If we're switching to `not in ...` then this becomes mute but I was hoping to keep to a convention within ganga so finding all of the `oDict` imports becomes easy and something we can _fix_ with a quick sed command or equivalent, this was not about caring for Python convention as it was for future maintenance (although a quick grep reveals this is minimal thankfully). If it stays I'll adopt the future naming scheme